docs/pages/components/button.md

```diff
@@ -240,3 +240,13 @@ Button with multiple actions
 <button class="fd-button--compact fd-button--menu">Compact</button>
 {% endcapture %}
 {% include display-component.html component=button-menu %}
+
+## Button With Text Overflow
+
+Content overflow in the button will be shown with ellipsis
+
+{% capture button %}
+<button class="fd-button">Default button with longer text</button>
```

Comment:
I don't see ellipsis in this example
<img width="511" alt="Screen Shot 2020-01-23 at 8 33 10 AM" src="https://user-images.githubusercontent.com/4380815/72988811-01681b00-3dbb-11ea-8d28-6efb80a195b6.png">

Reply:
Hi Deno, 
     As requested in the bug truncation is applied based on the parent container width.
For example.
When we decrease screen width in inspect mode mobile view or tab with or with lesser screen width we can see the truncation effect.

Reply:
Sure, In button.md I will remove the duplicate example and update with a different examples.

Reply:
> Hi Deno,
> As requested in the bug truncation is applied based on the parent container width.
> For example.
> When we decrease screen width in inspect mode mobile view or tab with or with lesser screen width we can see the truncation effect.

What is the value of adding another example if we cannot show it in the documentation.
Btw the menu button spec has more requirements in terms of `max-width`

Reply:
Added example with lesser parent width which shows the truncation.

Reply:
As per firori when we have a menu in the split button then the max-width is 12rem after that truncation has to be applied. For this, we have a bug.
https://github.com/SAP/fundamental-styles/issues/602.